glommio/src/timer/timer_impl.rs

```diff
@@ -881,23 +881,6 @@ mod test {
         });
     }
 
-    #[test]
-    fn basic_timer_action_instant_past_works() {
```

Comment:
this test cannot be deleted.

It is testing something important: that a timer that expired in the past will still trigger.

Reply:
Apologies, this test is passing just fine. I added it back!